### ISSUE TITLE
MOT label format inconsistence

### ISSUE THREAD

**Issue by NeoYY** [MrAsimZahid/Dataset-Format-Conversions#4]
Hi!

You extract top left coordinates, width, and height through
```
    x1 = row.iloc[0]['BB_LEFT']
    y1 = row.iloc[0]['BB_TOP']
    x2 = row.iloc[0]['BB_WIDTH']
    y2 = row.iloc[0]['BB_HEIGHT']
```
However, you consider x2 and y2 as maximum x and maximum y, which does not sound correct to me.

In addition, from the YOLOv5 repo, they state the label as 
`class x_center y_center width height`
but what you are using is 
`yolo_x, yolo_w, yolo_y, yolo_h`
Can you explain the above? Thanks!

**Comment by MrAsimZahid** [comment]
@NeoYY you are right. This is a logical error. Feel free to create a pull request. Thanks.
